### ISSUE TITLE
TimeOnly.ToClockNotation with german culture returns english like time

### ISSUE THREAD

**Issue by RenNagasaki** [Humanizr/Humanizer#1532]
If you use .ToClockNotation on a TimeOnly object created with german culture it still returns the english time definition.

Like: 08:30 becomes "half past acht". (The number is correctly translated but not the description of the time)
It should be either: "halb neun" or "acht uhr dreißig".
This goes for all variants of time. **:45/**:15

Using it with a time like: 8:37 returns "acht siebenunddreißig" 
but should be "acht uhr siebenunddreißig".
![grafik](https://github.com/user-attachments/assets/d320cb82-d085-40c5-b35e-70e49838677a)

I hope I made clear what I mean.

Kind regards,
Ren

**Comment by hangy** [comment]
True, this extension is currently only implemented for a handful of languages: https://github.com/Humanizr/Humanizer/blob/1dee67d72d53282aff76be8a2aaacc4a28ea6cbf/src/Humanizer/Configuration/TimeOnlyToClockNotationConvertersRegistry.cs#L8-L14
